execution_engine/src/shared/wasm_prep.rs

```diff
@@ -608,4 +609,48 @@ mod tests {
             error,
         );
     }
+
+    #[test]
+    fn should_not_accept_multi_value_proposal_wasm() {
+        let module_bytes = {
+            let mut module = walrus::Module::with_config(ModuleConfig::new());
+
+            let _memory_id = module.memories.add_local(false, 11, None);
+
+            // fn func_with_multi_value() -> (i32, i64) {
```

Comment:
There are a few commented lines in this function which look like they should be deleted.